src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,22 @@
 import React from 'react';
+import { Comments } from '../../types/Comments';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type CommentTypes = {
+  comment: Comments
```

Comment:
Why comment has type Comments?
```suggestion
  comment: Comment;
```